apps/system/utils/Kconfig

```diff
@@ -50,6 +50,43 @@ config NET_PING_CMD_ICOUNT
 
 endif #NET_CMDS
 
+config ENABLE_CPULOAD_CMD
+	bool "cpuload"
+	default y
+	depends on SCHED_CPULOAD
+	---help---
+		Display current cpuload or start/stop CPU load monitor.
+		CPU load monitor is a daemon that will periodically assess
+		CPU usage by all live tasks and threads in the system.
+
+if ENABLE_CPULOAD_CMD
+config ENABLE_CPULOAD_MONITOR
+	bool "cpuload monitor"
+	default y
+	depends on SCHED_CPULOAD && !CONFIG_DISABLE_PTHREAD && !CONFIG_DISABLE_SIGNALS
```

Comment:
Checking SCHED_CPULOAD is duplicated. That is checked on ENABLE_CPULOAD_CMD already. Let's remove it from here.